edb/schema/delta.py

```diff
@@ -933,7 +933,8 @@ class ObjectCommand(
     scls: so.Object_T
     _delta_action: ClassVar[str]
     _schema_metaclass: ClassVar[Optional[Type[so.Object_T]]]
-    astnode: ClassVar[Type[qlast.DDLOperation]]
+    astnode: ClassVar[Union[Type[qlast.DDLOperation],
+                            List[Type[qlast.DDLOperation]]]]
```

Comment:
Inside `constraints.py`, astnode class property is used to specify a list of `DDLOperation` types, rather than a single type. So, this change to delta.py looks correct as it describes the reality of the other module.

However, this change creates complications inside delta.py (TODO you can see below). Can you please recommend the best direction?